tests/zfs-tests/tests/functional/raidz/raidz_003_pos.ksh

```diff
@@ -36,6 +36,6 @@
 #	runtime might be longer.
 #
 
-log_must raidz_test -S -e -t 60
+log_must raidz_test -S -e -t 300
```

Comment:
I'm a little reluctant to increase the runtime of the test suite by 4 minutes here, and 4 more minutes in `raidz_004_pos`, unless we have a good reason to.

Reply:
I agree. As a compromise can we distill it down to one test that randomly picks one of the three flavors?
```
tests/zfs-tests/tests/functional/raidz/raidz_002_pos.ksh:log_must raidz_test -S -t 300
tests/zfs-tests/tests/functional/raidz/raidz_003_pos.ksh:log_must raidz_test -S -e -t 300
tests/zfs-tests/tests/functional/raidz/raidz_004_pos.ksh:log_must raidz_test -S -e -r 0 -t 300
```

Reply:
Sure, that works for me.

Reply:
Made the change.